tests/rule_based_profiler/data_assistant/test_volume_data_assistant.py

```diff
@@ -3213,3 +3214,90 @@ def test_volume_data_assistant_plot_custom_theme_overrides(
         chart.config.axisX["titleColor"] == x_axis_title_color
         for chart in plot_result.charts
     )
+
+
+def test_volume_data_assistant_plot_return_tooltip(
+    volume_data_assistant_result: VolumeDataAssistantResult,
+) -> None:
+    plot_result: PlotResult = volume_data_assistant_result.plot(prescriptive=True)
+
+    expected_tooltip: List[alt.Tooltip] = [
+        alt.Tooltip(
+            **{
+                "field": "column",
+                "format": "",
+                "title": "Column",
+                "type": AltairDataTypes.NOMINAL.value,
+            }
+        ),
```

Comment:
Nitpick - any reason for **kwarg unpacking when we can use named args to instantiate without the need for a dict?

Reply:
It's a lot less work to do this when you copy/paste configs from the debugger.